### PR TITLE
Add zulu.xml

### PR DESCRIPTION
**Pull request by RoshanRathod0137**

### CODE REVIEW

**Comment by chcg** [comment]
Additionally an entry in:
PowerEditor/src/localizationString.h
is needed and for installation/uninstallation:
PowerEditor/installer/nsisInclude/langs4Npp.nsh

**Comment by RoshanRathod0137** [comment]
We have added Additional  entry in:PowerEditor/src/localizationString.h
and for installation/uninstallation: PowerEditor/installer/nsisInclude/langs4Npp.nsh

**Comment by SinghRajenM** [comment]
I'm away and don't have access to my dev pc. I can check once I come back.

File: PowerEditor/installer/nativeLang/ZULU.xml [hunk 1]
```diff
@@ -0,0 +1,1183 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/ZULU.xml]: file name should be all in lower case

File: PowerEditor/installer/nsisInclude/langs4Npp.nsh [hunk 2]
```diff
@@ -286,6 +286,9 @@ SectionGroup "Localization" localization
 	${MementoUnselectedSection} "Piglatin" piglatin
 		CopyFiles "$PLUGINSDIR\nppLocalization\piglatin.xml" "$INSTDIR\localization\piglatin.xml"
 	${MementoSectionEnd}
+        ${MementoUnselectedSection} "ZULU" ZULU
```
Comment (donho) [inline PowerEditor/installer/nsisInclude/langs4Npp.nsh]: Respect the convention: "Zulu" zulu

File: PowerEditor/installer/nsisInclude/langs4Npp.nsh [hunk 3]
```diff
@@ -547,4 +550,7 @@ SectionGroup un.localization
 	Section un.piglatin
 		Delete "$INSTDIR\localization\piglatin.xml"
 	SectionEnd
+        Section un.ZULU
```
Comment (donho) [inline PowerEditor/installer/nsisInclude/langs4Npp.nsh]: un.zulu

File: PowerEditor/src/localizationString.h [hunk 4]
```diff
@@ -115,5 +115,6 @@ LocalizationSwitcher::LocalizationDefinition localizationDefs[] =
 	{TEXT("Corsu"), TEXT("corsican.xml")},
 	{TEXT("Brezhoneg"), TEXT("breton.xml")},
 	{TEXT("کوردی‬"), TEXT("kurdish.xml")},
-	{TEXT("Pig latin"), TEXT("piglatin.xml")}
-};
\ No newline at end of file
+	{TEXT("Pig latin"), TEXT("piglatin.xml")},
+        {TEXT("ZULU"), TEXT("ZULU.xml")}
```
Comment (donho) [inline PowerEditor/src/localizationString.h]: TEXT("Zulu"), TEXT("zulu.xml")}

**Comment by RoshanRathod0137** [comment]
@donho i have done with your given changes with lower case and naming convention

**Comment by donho** [comment]
@RoshanRathod0137  There are 2 PR opened currently.
Please close 1 of these 2 so I know which one I should check.

**Pull request closed without merging**